From 255ef81b3ece85da9fc1941e653d6d6c73144327 Mon Sep 17 00:00:00 2001
From: Frank Bauernoeppel <frank@bauernoeppel.de>
Date: Wed, 18 Sep 2013 21:40:49 +0200
Subject: [PATCH 4/4] rootfs is loaded from /dev/mmcblk0p1 (but freezes in
 systemd)

---
 arch/arm/tools/mach-types              |    2 +-

diff --git a/arch/arm/tools/mach-types b/arch/arm/tools/mach-types
index c241052..729592e 100644
--- a/arch/arm/tools/mach-types
+++ b/arch/arm/tools/mach-types
@@ -1125,4 +1125,4 @@ mx6q_arm2		MACH_MX6Q_ARM2		MX6Q_ARM2		3837
 mx6sl_arm2		MACH_MX6SL_ARM2		MX6SL_ARM2		4091
 mx6q_hdmidongle		MACH_MX6Q_HDMIDONGLE    MX6Q_HDMIDONGLE		4284
 mx6sl_evk		MACH_MX6SL_EVK		MX6SL_EVK		4307
-
+mx6_dres0280		MACH_MX6_DRES0280	MX6_DRES0280		9999
-- 
1.7.9.5

